1.8/usage/cli/index.md

```diff
@@ -37,21 +37,21 @@ Get detailed command description with 'dcos <command> --help'.
 
 These environment variables are supported by the DC/OS CLI and can be set dynamically.
 
-#### `DCOS_CONFIG` 
-Set the path to the DC/OS configuration file. By default, this variable is set to `DCOS_CONFIG=/<home-directory>/.dcos/dcos.toml`. For example, if you moved your DC/OS configuration file to `/home/jdoe/config/` you can specify this command:
+#### DCOS_CONFIG
+Specifies the path to the DC/OS configuration file. By default, this variable is set to `DCOS_CONFIG=/<home-directory>/.dcos/dcos.toml`. For example, if you moved your DC/OS configuration file to `/home/jdoe/config/` you can specify this command:
 
 ```bash
 export DCOS_CONFIG=/home/jdoe/config/dcos.toml
 ```
     
-#### `DCOS_SSL_VERIFY` 
-Indicates whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). By default, this is variable is set to `true`. Setting this environment variable is equivalent to setting the `core.ssl_config` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
+#### DCOS_SSL_VERIFY (required)
+Specifies whether to verify SSL certificates for HTTPS (`true`) or set the path to the SSL certificates (`false`). You must set this variable manually. Setting this environment variable is equivalent to setting the `dcos config set core.ssl_verify` option in the DC/OS configuration [file](#configuration-files). For example, to indicate that you want to set the path to SSL certificates:
```

Comment:
Changed to `Indicates whether to verify SSL certificates or set the path to the SSL certificates.`